Makefile.in

```diff
@@ -415,22 +415,21 @@ testobsoletes:
             `date -u +dev/log/testobsoletesN1_%Y-%m-%d-%H-%M`; rm testobsoletesN1.in )
 	( rm wsp.g )
 
-cygwin:	extern
-	cnf/mkversionheader.sh bin/$(GAPARCH)/gap_version.h
-	( cd bin/$(GAPARCH) ; $(MAKE) gapdll ABI='$(ABI)' CC='$(CC)' GMP_VER='$(GMP_VER)' GMP_CFLAGS='$(GMP_CFLAGS)' GMP_LIBS='$(GMP_LIBS)' CONFIGNAME='$(CONFIGNAME)' )
-	chmod +x bin/gap-$(CONFIGNAME).sh
+ifeq ($(shell uname -o), Cygwin)
+default: cygwin
+endif
+
+cygwin: compile
 	chmod +x bin/*.bat
-	cp bin/$(GAPARCH)/gapw95.exe bin
-	cp bin/$(GAPARCH)/gap.dll bin
+	chmod +x cnf/instcygwinterminfo.sh
+	cnf/instcygwinterminfo.sh
+	( if which peflags ; then peflags --cygwin-heap=2048 bin/$(GAPARCH)/gap.exe ; fi )
 	cp /bin/cygncurses-10.dll /bin/cygncursesw-10.dll \
```

Comment:
Sorry, hadn't pushed everything, now sorted out (well, will flatten out commits at some point).

A bunch of whitespace diffs has snuck in, but I'm feeling too lazy to take them out.